### ISSUE TITLE
Upload 2 big files

### ISSUE THREAD

**Issue by timediv** [balderdashy/skipper#75]
Uploading two big files with req.file('a') and req.file('b') results in one timing out with

verbose: Error: ETIMEOUT: An Upstream (`b`) timed out waiting for file(s). No files were sent after waiting 500ms.

I suspect one of the files times out while the other one is being uploaded. If I need to reconfigure the 500ms, where do I need to put the config file and what should it look like?

I'm using sails 0.10.5

Same problem occurs with latest beta sails and skipper 0.5.5 just with a higher ETIMEOUT of 10000ms.

**Comment by metzuda** [comment]
If I understand you correctly, you're attempting to upload to large files on the same request, and trying to upload them at the same time. 

Is it possible to upload them one after the other in series? Uploading them in series should take roughly the same amount of time, I would think.

**Comment by sailsbot** [comment]
Thanks for posting, @timediv.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by timediv** [comment]
Guten Tag,

leider sind wir bis zum 8.10 im Urlaub. Wir melden uns bei Ihnen zurück!

Vielen Dank!

mit freundlichen Grüßen

Louis Kirsch
